### PR TITLE
[Proposal] SF-0010 Rename accepted proposal (was previously NNNN)

### PR DESCRIPTION
**Pull request by hristost**

### CODE REVIEW

**Comment by hristost** [comment]
@swift-ci please test

**Comment by hristost** [comment]
@swift-ci please test

File: Proposals/0010-calendar-recurrence-rule-end-count-and-date.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Extending `Calendar.RecurrenceRule.End`
 
 
-* Proposal: SF-NNNN
+* Proposal: SF-0010
 * Author(s): Hristo Staykov <hstaykov@apple.com>
 * Review Manager: [Tina Liu](https://github.com/itingliu)
 * Status: **Active review**
```
Comment (itingliu) [inline Proposals/0010-calendar-recurrence-rule-end-count-and-date.md:7]:
```suggestion
* Status: **Implemented**
```

While you're here might as well update the status

**Comment by itingliu** [comment]
@swift-ci please test

**Review by itingliu — APPROVED** [review]

**Pull request merged**